test/officerManager.js

```diff
@@ -0,0 +1,336 @@
+/* global describe it before after */
+// During the test the env variable is set to test
+process.env.NODE_ENV = 'test'
+const OfficerManager = require('../api/models/OfficerManager')
+const User = require('../api/models/User')
+// Require the dev-dependencies
+const chai = require('chai')
+const chaiHttp = require('chai-http')
+const server = require('../server')
+let serverInstance = null
+let app = null
+const expect = chai.expect
+// tools for testing
+const tools = require('../util/testing-utils/tools.js')
+
+chai.should()
+chai.use(chaiHttp)
+
+function initializeServer () {
```

Comment:
In a seperate PR, we should throw `initializeServer` and `terminateServer` this into tools. I will open an issue for it.

Reply:
See #150

Reply:
can you work on that. I tried to pull them over to util but having issues with the variables